bool is3d

int32 job_number
int32 obj_id
char bin_loc
float32 confidence

float32 centroid_x
float32 centroid_y
float32 centroid_z

float32 angle_ma
float32 angle_mi
